evm_loader/lib/src/account_storage_tests.rs

```diff
@@ -112,6 +113,50 @@ where
 
     Ok(action(&balance_account?))
 }
+
+async fn check_contract_code<T: rpc::Rpc>(
+    storage: &EmulatorAccountStorage<'_, T>,
+    address: Address,
+    code: Vec<u8>,
+) -> NeonResult<bool> {
+    let mut account_data = storage
+        .get_contract_account(address)
+        .await
+        .map_err(map_neon_error)?
+        .borrow_mut();
+
+    let contract =
+        ContractAccount::from_account(storage.program_id(), account_data.into_account_info())?;
+
+    let is_equal = hex::encode(&*contract.code()) == hex::encode(code);
```

Comment:
You don't need to convert to hex to compare 2 byte arrays.

Reply:
done